### ISSUE TITLE
Util.isNumber falsely says that version-like strings like "1.2.3" is a number

### ISSUE THREAD

**Issue by martin-g** [kairosdb/kairosdb#468]
We have errors like:

```
07:14:11.850 [qtp467313213-40] ERROR [MetricsResource.java:396] - Failed to add metric.
java.lang.NumberFormatException: multiple points
    at sun.misc.FloatingDecimal.readJavaFormatString(FloatingDecimal.java:1914)
    at sun.misc.FloatingDecimal.parseDouble(FloatingDecimal.java:110)
    at java.lang.Double.parseDouble(Double.java:538)
    at com.google.gson.JsonPrimitive.getAsDouble(JsonPrimitive.java:193)
    at org.kairosdb.core.datapoints.DoubleDataPointFactoryImpl.getDataPoint(DoubleDataPointFactoryImpl.java:40)
    at org.kairosdb.core.GuiceKairosDataPointFactory.createDataPoint(GuiceKairosDataPointFactory.java:111)
    at org.kairosdb.core.http.rest.json.DataPointsParser.validateAndAddDataPoints(DataPointsParser.java:384)
    at org.kairosdb.core.http.rest.json.DataPointsParser.parse(DataPointsParser.java:94)
    at org.kairosdb.core.http.rest.MetricsResource.add(MetricsResource.java:372)
    ...
```

I think it is caused by https://github.com/kairosdb/kairosdb/blob/ac248d35050393191e99779a7addaf9334900d89/src/main/java/org/kairosdb/util/Util.java#L229-L257 which is used at https://github.com/kairosdb/kairosdb/blob/ac248d35050393191e99779a7addaf9334900d89/src/main/java/org/kairosdb/core/http/rest/json/DataPointsParser.java#L237
At the moment `isNumber("1.2.3")` returns `true` and later `Double.parseDouble()` fails on it.

**Comment by brianhks** [comment]
Good catch we need to fix this.

**Comment by jsabin** [comment]
Fixed in commit 3427e76052b5f442fb4abf8e0dbf24bbd0e67348.
